### PR TITLE
Adds Airbase to the list

### PR DESCRIPTION
**Pull request by anmdotdev**
## Add Airbase

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [x] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

**Comment by anmdotdev** [comment]
Can we merge this?

File: README.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Also check out [No Whiteboards](https://nowhiteboards.io) to search for jobs at
 - [AgileMD](https://angel.co/agilemd/jobs) | San Francisco, CA | Takehome project
 - [Agoda](https://careersatagoda.com/departments/technology) | Bangkok, Thailand | Take-home project, then a discussion onsite round.
 - [AImotive](https://aimotive.com/career/) | Budapest, Hungary | Small take-home project, then in-person interview about technical background
+- [Airbase](https://www.airbase.com/careers) | San Francisco, CA; India; Remote | Intro Call with HR or EMs, small take-home project, two tech rounds with Airbase Engineers (based on real world engineering problems), followed by a final values round with the Founder or Director of Engg.
```
Comment (poteto) [inline README.md:59]: Could you add where in India?

Comment (anmdotdev) [reply README.md:59]: It hires anywhere in India. Its a distributed, fully remote company.

Comment (anmdotdev) [reply README.md]: @poteto Added Bangalore, India.

**Pull request merged**